packages/back-end/src/models/VisualChangesetModel.ts

```diff
@@ -505,3 +505,17 @@ export const deleteVisualChangesetById = async ({
     visualChangeset,
   });
 };
+
+export const findExperimentByVisualChangesetId = async (
+  visualChangesetId: string,
+  organization: string
+): Promise<ExperimentInterface | null> => {
+  const visualChangeset = await findVisualChangesetById(
+    visualChangesetId,
+    organization
+  );
+
+  if (!visualChangeset) return null;
+
+  return getExperimentById(organization, visualChangeset.experiment);
```

Comment:
The method signature for `getExperimentById` has changed to work with a Context object so this will need to be updated

Reply:
Will update.

Interesting that there are no merge conflicts. I'm guessing if this were merged, it would have failed on main.